client/my-sites/patterns/components/get-started/index.tsx

```diff
@@ -0,0 +1,62 @@
+import { Button } from '@wordpress/components';
+import ImgBlockPatterns from 'calypso/my-sites/patterns/components/get-started/images/block-patterns.svg';
+import ImgHomePage from 'calypso/my-sites/patterns/components/get-started/images/home-page.svg';
+import ImgPageLayouts from 'calypso/my-sites/patterns/components/get-started/images/page-layouts.svg';
+import { PatternsSection } from 'calypso/my-sites/patterns/components/section';
+
+import './style.scss';
+
+export const PatternsGetStarted = () => {
+	return (
+		<PatternsSection
+			title="Get started with patterns"
+			description="Our favorite how-to guides to get you started with patterns."
+			theme="dark"
+			bodyFullWidth
+		>
+			<div className="patterns-get-started--body">
```

Comment:
```suggestion
			<div className="patterns-get-started__body">
```
For class names, we should use `__` to delineate “blocks” and "elements", and `_` to delineate “elements” and “modifiers”.

I.e., `patterns-get-started--body` should be `patterns-get-started__body` (and if it had a modifier, it should be `patterns-get-started__body_blue`).

This applies to the whole PR, but I'll leave just one comment.

I mentioned this in Slack as well p1709203022906369/1709186405.537729-slack-C06ELHR6L9J